src/hotspot/share/runtime/objectMonitor.cpp

```diff
@@ -1011,11 +1008,6 @@ void ObjectMonitor::ReenterI(JavaThread* current, ObjectWaiter* currentNode) {
     }
 
     // The lock is still contested.
-    // Keep a tally of the # of futile wakeups.
-    // Note that the counter is not protected by a lock or updated by atomics.
-    // That is by design - we trade "lossy" counters which are exposed to
-    // races during updates for a lower probe effect.
```

Comment:
The same comment block needs deleting at line 874 too.

Reply:
Actually, I think the comment blocks are referring to the `OM_PERFDATA_OP(FutileWakeups, inc());` calls not the `nWakeups` local variable.